src/Plugin/Validation/Constraint/InstagramEmbedCodeConstraintValidator.php

```diff
@@ -19,14 +19,14 @@ class InstagramEmbedCodeConstraintValidator extends ConstraintValidator {
   /**
    * {@inheritdoc}
    */
-  public function validate($entity, Constraint $constraint) {
-    if (!isset($entity)) {
+  public function validate($value, Constraint $constraint) {
+    if (!isset($value)) {
       return;
     }
 
     $matches = [];
     foreach (Instagram::$validationRegexp as $pattern => $key) {
-      if (preg_match($pattern, $entity->value, $item_matches)) {
+      if (preg_match($pattern, $value, $item_matches)) {
```

Comment:
I can add test coverage, but I tried this in devel/php and only got it to produce no violations when $value was a string:

```php
$embed_code = 'https://twitter.com/webchick/status/672110599497617408';
$definition = \Drupal::typedDataManager()->createDataDefinition('string');
$definition->addConstraint('TweetEmbedCode');
$value = \Drupal::typedDataManager()->create($definition, $embed_code);
$violations = $value->validate();
dpm($violations->count());
```